libs/SalesforceFileLogger/SalesforceFileLogger/Classes/Logger/SFSDKFileLogger.h

```diff
@@ -0,0 +1,56 @@
+/*
+ SFSDKFileLogger.h
+ SalesforceAnalytics
```

Comment:
Might want to replace `SalesforceAnalytics` here with `SalesforceFileLogger` instead, and in other header comments.